apps/authentication-service/cypress/e2e/clients.cy.ts

```diff
@@ -220,4 +258,38 @@ describe("Client API", () => {
       expect(response.status).to.eq(403);
     });
   });
+
+  it("creates a new client, accesses institution cache endpoint, and deletes the client", () => {
+    cy.request({
+      method: "POST",
+      url: keysUrl,
+      headers: {
+        ContentType: "application/json",
+        Authorization: `Bearer ${accessToken}`,
+      },
+    }).then(() => {
+      cy.request({
+        url: `http://localhost:8088/institutions/cacheList`,
+        method: "GET",
+        headers: {
+          Authorization: `Bearer ${accessTokenWidget}`,
```

Comment:
This isn't testing that the application you just created has access to what it's supposed to. This is using a token from something that is kind of fake. This doesn't give me confidence that the create keys endpoint creates an application with the access that it needs.

Reply:
Okay, new code that actually tests with the newly created client.